tests/Consumer/AbstractConsumerTest.php

```diff
@@ -0,0 +1,102 @@
+<?php
+namespace Tests\AppBundle\Consumer;
+
+use Doctrine\Bundle\DoctrineBundle\Registry;
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Persistence\ObjectManager;
+use OldSound\RabbitMqBundle\RabbitMq\ConsumerInterface;
+use PhpAmqpLib\Message\AMQPMessage;
+use PHPUnit\Framework\TestCase;
+use Psr\Log\LoggerInterface;
+use Symfony\Component\Validator\ConstraintViolationInterface;
+use Symfony\Component\Validator\ConstraintViolationListInterface;
+use Symfony\Component\Validator\Validator\ValidatorInterface;
+
+class AbstractConsumerTest extends TestCase
+{
+    const CLASS_NAME = 'AppBundle\Consumer\AbstractConsumer';
+
+    /**
+     * @var \PHPUnit_Framework_MockObject_MockObject
```

Comment:
`\PHPUnit_Framework_MockObject_MockObject|ValidatorInterface`, same for others below.